src/software/ai/hl/stp/tactic/penalty_kick/simulated_penalty_kick_tactic_test.cpp

```diff
@@ -0,0 +1,101 @@
+#include <gtest/gtest.h>
+
+#include "software/simulated_tests/simulated_tactic_test_fixture.h"
+#include "software/ai/hl/stp/tactic/penalty_kick/penalty_kick_tactic.h"
+#include "software/simulated_tests/non_terminating_validation_functions/ball_in_play_or_scored_validation.h"
+#include "software/simulated_tests/non_terminating_validation_functions/ball_never_moves_backward_validation.h"
+#include "software/simulated_tests/non_terminating_validation_functions/robot_not_excessively_dribbling_validation.h"
+#include "software/simulated_tests/terminating_validation_functions/friendly_scored_validation.h"
+#include "software/test_util/test_util.h"
+#include "software/world/field.h"
+
+//goalie
+class SimulatedPenaltyKickTacticTest
+    : public SimulatedTacticTestFixture,
+      public ::testing::WithParamInterface<std::tuple<RobotStateWithId>>
+{
+    protected :
+        void SetUp() override
+        {
+            setBallState(BallState(field().friendlyPenaltyMark(), Vector(0, 0)));
+            Point initial_position = field().friendlyPenaltyMark() + Vector(-0.1, 0);
+            addFriendlyRobots(TestUtil::createStationaryRobotStatesWithId({initial_position}));
+        }
+};
+
+// TEST_P(SimulatedPenaltyKickTacticTest, penalty_kick_test)
```

Comment:
just wondering why `TEST_P` is commented out here.

Reply:
same thing as above, it's active now